test/factories/forms.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :form do
+    name { Faker::Lorem.sentence }
+    description { Faker::Lorem.sentence }
+    title { name }
+    instructions { Faker::Lorem.paragraph }
+    organization
```

Comment:
I think it would be best to remove the `organization` association from the factory. This app is using `ActsAsTenant` to enforce tenant assignment to the current, active tenant (which is set by default in the `test_helper`). Because of this, giving an `organization` association in the factory is misleading as whatever option is passed will be overwritten by `ActsAsTenant`.

Reply:
Done!  Thanks!